ci/values/k3s.yaml

```diff
@@ -9,6 +9,9 @@ ingress:
 web:
   secureCookies: false
 
+internalMetrics:
```

Comment:
wrong indentation? It should be `web.internalMetrics` right now it seems to be at the top level

Reply:
Thanks! This caused me trouble without knowing about this :sweat_smile: